refinery/ui/source/js/dashboard/controllers/vis-wrapper.js

```diff
@@ -15,51 +31,191 @@ function VisWrapperCtrl ($q, pubSub, dashboardVisData) {
   // Remix root nodes are a collection of nodes that act as meaningful entry
   // points across different ontologies in regards to browsing.
   self.remixRoots = [
-    'http://purl.obolibrary.org/obo/BTO_0002666',
-    'http://purl.obolibrary.org/obo/BTO_0000088',
-    'http://purl.obolibrary.org/obo/BTO_0000421',
-    'http://purl.obolibrary.org/obo/BTO_0000174',
-    'http://purl.obolibrary.org/obo/BTO_0000522',
-    'http://purl.obolibrary.org/obo/BTO_0000282',
-    'http://purl.obolibrary.org/obo/BTO_0000570',
-    'http://purl.obolibrary.org/obo/BTO_0005810',
-    'http://purl.obolibrary.org/obo/BTO_0000634',
+    'http://bioontology.org/ontologies/BiomedicalResourceOntology.owl#Information_Resource',
```

Comment:
Why? They are only used in one place and don't have to be shared across multiple components.